localhost/mac/localhost/telemetry/installer/mac/functions.bash

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+#################################################################################################################################
+################################################### Installer Mac Functions #####################################################
+#################################################################################################################################
+
+# Gets which products were selected (logs/metrics)
+# Output:
+#   is_logs_option_selected - Tells if logs option was selected (true/false)
+#   logs_params - The logs params if logs option was selected
+#   is_metrics_option_selected - Tells if metrics option was selected (true/false)
+#   metrics_params - The metrics params if metrics option was selected
+# Error:
+#   Exit Code 1
+function get_which_products_were_selected () {
+    write_log "INFO" "Getting which products were selected ..."
```

Comment:
please replace to "Checking"

Reply:
In this function I am more getting the selected products (insert them into variables) rather than checking...

Reply:
The syntax is not correct 
Maybe 
Getting the selected products?

Reply:
done